[bjnp] create_broadcast_socket: ERROR - bind socket to local address failed - Cannot assign requested address
[bjnp] create_broadcast_socket: ERROR - bind socket to local address failed - Cannot assign requested address
Traceback (most recent call last):
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/product_hub.py", line 47, in edit_product_clicked
    p.select_product ()
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/products.py", line 665, in select_product
    "FROM products WHERE id = (%s)", (self.product_id,))
psycopg2.ProgrammingError: column "invoice_serial_numbers" does not exist
LINE 1: ...e_account, income_account, manufacturer_sku, job, invoice_se...
                                                             ^
HINT:  Perhaps you meant to reference the column "products.enforce_serial_numbers".

Traceback (most recent call last):
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/products.py", line 309, in focus_in_populate_comboboxes
    self.populate_account_combos ()
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/products.py", line 315, in populate_account_combos
    self.cursor.execute("SELECT id, name FROM tax_rates ORDER BY name")
psycopg2.InternalError: current transaction is aborted, commands ignored until end of transaction block

Traceback (most recent call last):
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/product_hub.py", line 45, in edit_product_clicked
    p = products.ProductsGUI (self.main)
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/products.py", line 84, in __init__
    self.new_product()
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/products.py", line 892, in new_product
    self.cursor.execute("SELECT id FROM tax_rates WHERE standard = True ")
psycopg2.InternalError: current transaction is aborted, commands ignored until end of transaction block

Traceback (most recent call last):
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/products.py", line 309, in focus_in_populate_comboboxes
    self.populate_account_combos ()
  File "/media/reuben/nas/pygtk_posting-0.3.1/src/products.py", line 315, in populate_account_combos
    self.cursor.execute("SELECT id, name FROM tax_rates ORDER BY name")
psycopg2.InternalError: current transaction is aborted, commands ignored until end of transaction block

Traceback (most recent call last):
  File "./src/pygtk_posting.py", line 456, in destroy
    self.cursor.execute("UNLISTEN products")
psycopg2.InternalError: current transaction is aborted, commands ignored until end of transaction block

